datastore/longhorn.go

```diff
@@ -402,21 +415,35 @@ func (s *DataStore) ValidateSetting(name, value string) (err error) {
 	return nil
 }
 
-func (s *DataStore) ValidateV2DataEngine(v2DataEngineEnabled bool) error {
-	if !v2DataEngineEnabled {
-		allV2VolumesDetached, err := s.AreAllV2VolumesDetached()
+func (s *DataStore) ValidateV1DataEngine(dataEngineEnabled bool) error {
```

Comment:
nit: `ValidateV1DataEngineEnabled`

the same for v2.

Reply:
Renamed. Thank you.